src/scripting/context.rs

```diff
@@ -93,8 +100,8 @@ impl RowDistributionPreset {
         }
     }
 
-    pub async fn get_partition_idx(&self, idx: u64) -> u64 {
-        self._get_partition_idx(
+    pub async fn get_partition(&self, idx: u64) -> (u64, u64) {
```

Comment:
helpful to add small docstring what is returned in tuple (previously could guess it from fn name)